scripts/ci/build.sh

```diff
@@ -26,9 +26,7 @@ cp $(cd $(dirname $0); pwd)/app/* $app_dir
 
 ##############################################
 # Update version strings
-if $TRAVIS; then
-    $(cd $(dirname $0); pwd)/version.sh $1
-fi
+. $(cd $(dirname $0); pwd)/version.sh $1
```

Comment:
Nit: I personally prefer to assign more meaningful variable names at the top of my script.

Reply:
yo. will do in next change.